### PR TITLE
Localize refreshed VPN pages

### PR DESCRIPTION
**Pull request by craigcook**
## One-line summary

Putting the hard-coded English text from the refreshed VPN pages into Fluent files to start l10n.

Landing page: http://localhost:8000/products/vpn/
Pricing page: http://localhost:8000/products/vpn/pricing/
Features page: http://localhost:8000/products/vpn/features/

### CODE REVIEW

**Comment by craigcook** [comment]
The landing page and pricing page are pre-existing, so should only show the refresh template if the Fluent file is active.

The features page is new so it doesn't need any special view logic.

**Comment by reemhamz** [comment]
Note that I took this PR over since Craig is working on some other stuff

File: bedrock/products/views.py [hunk 1]
```diff
@@ -125,7 +121,7 @@ def vpn_pricing_page(request):
 @require_safe
 def vpn_features_page(request):
     template_name = "products/vpn/features.html"
-    ftl_files = ["products/vpn/shared"]
+    ftl_files = ["products/vpn/shared", "products/vpn/features"]
```
Comment (alexgibson) [inline bedrock/products/views.py:124]: Issue: we should use `products/vpn/features.ftl` as the activation file, so this should be ordered the other way around:
```suggestion
    ftl_files = ["products/vpn/features", "products/vpn/shared"]
```

File: bedrock/products/views.py [hunk 2]
```diff
@@ -97,14 +88,19 @@ def vpn_landing_page(request):
...
     variant = request.GET.get("v", None)
 
+    if ftl_file_is_active("products/vpn/pricing-2023"):
+        template_name = "products/vpn/pricing-refresh.html"
+    else:
+        template_name = "products/vpn/pricing.html"
+
     # ensure variant matches pre-defined value
     if variant not in ["1", "2"]:
         variant = None
```
Comment (alexgibson) [inline bedrock/products/views.py:106]: Question: do we still need the `variant` logic for this view?

File: l10n/en/products/vpn/features.ftl [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+#   $devices (number) - number of devices users can connect to VPN
+vpn-features-connect-up-to = Connect up to { $devices } devices
+vpn-features-supported-platforms = Supported on Windows, macOS, Android, iOS and Linux operating systems.
+vpn-features-no-bandwidth = No bandwidth restrictions or throttling
+vpn-features-including-no-data = Including no data cap or speed limit.
+vpn-features-fast-network = Fast network speeds even while gaming
+
+# Variables
+#   $wireguard (url) - link to https://mullvad.net/help/why-wireguard/
+vpn-features-mozilla-vpn-uses-wireguard =  { -brand-name-mozilla-vpn } uses <a { $wireguard }>Wireguard</a>™, one of the most performant VPN protocols.
```
Comment (alexgibson) [inline l10n/en/products/vpn/features.ftl:32]: Nit: remove extra space
```suggestion
vpn-features-mozilla-vpn-uses-wireguard = { -brand-name-mozilla-vpn } uses <a { $wireguard }>Wireguard</a>™, one of the most performant VPN protocols.
```

File: l10n/en/products/vpn/features.ftl [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+vpn-features-combine-mozilla-vpn-with-containers = Combine { -brand-name-mozilla-vpn } with the Multi-Account Containers { -brand-name-firefox } extension andset different VPN locations per { -brand-name-firefox } tab. <a { $containers }>Learn how</a>.
+
+vpn-features-trustworthy = Trustworthy
+vpn-features-money-back = 30-day money-back guarantee
+vpn-features-plus-customer-support = Plus 24/7 customer support.
+vpn-features-we-never-log = We never log, track or share your network data
+
+# Variables
+#   $privacy (url) - link to https://www.mozilla.org/privacy/subscription-services/
+vpn-features-simply-put-we-dont = Simply put, we don’t collect your personal browsing information.Here’s our <a { $privacy }>easy-to-read privacy policy</a>.
```
Comment (alexgibson) [inline l10n/en/products/vpn/features.ftl:70]: Nit: add space after period
```suggestion
vpn-features-simply-put-we-dont = Simply put, we don’t collect your personal browsing information. Here’s our <a { $privacy }>easy-to-read privacy policy</a>.
```

File: l10n/en/products/vpn/features.ftl [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+vpn-features-built-transparently = Built transparently in open source
+
+# Variables
+#   $github (url) - link to https://github.com/mozilla-mobile/mozilla-vpn-client
+vpn-features-made-with-open-source-code =  { -brand-name-mozilla-vpn } is made with open-source code, meaning that all thecode is publicly accessible. See our <a { $github }>Github repository</a>.
```
Comment (alexgibson) [inline l10n/en/products/vpn/features.ftl:76]: Nit: remove extra space
```suggestion
vpn-features-made-with-open-source-code = { -brand-name-mozilla-vpn } is made with open-source code, meaning that all thecode is publicly accessible. See our <a { $github }>Github repository</a>.
```

File: l10n/en/products/vpn/pricing-2023.ftl [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+vpn-pricing-faqs = FAQs
+vpn-pricing-refund-policy = What is { -brand-name-mozilla-vpn }’s refund policy?
+vpn-pricing-the-first-time-you = The first time you subscribe to { -brand-name-mozilla-vpn } through { -brand-name-mozilla }’s website, if you cancel your account within the first 30 days, you may request a refund and { -brand-name-mozilla } will refund your first subscription term.
+vpn-pricing-if-you-purchased = If you purchased your subscription through in-app purchase from the Apple App Store or the Google Play Store, your payment is subject to the terms and conditions of the App Store. You must direct any billing and refund inquiries for such purchases to Apple or Google, as appropriate.
+
+vpn-pricing-what-information = What information does { -brand-name-mozilla-vpn } keep?
+
+# Variables
+#   $principles (url) - link to https://www.mozilla.org/en-US/privacy/principles/
+#   $notice (url) - link to https://www.mozilla.org/en-US/privacy/subscription-services/
```
Comment (alexgibson) [inline l10n/en/products/vpn/pricing-2023.ftl:41]: Nit: remove locale from URLs
```suggestion
#   $principles (url) - link to https://www.mozilla.org/privacy/principles/
#   $notice (url) - link to https://www.mozilla.org/privacy/subscription-services/
```

File: l10n/en/products/vpn/pricing-2023.ftl [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+vpn-pricing-we-adhere-strictly = We adhere strictly to { -brand-name-mozilla }’s <a { $principles }">Data Privacy Principles</a>. We only collect data required to keep { -brand-name-mozilla-vpn } operational and improve the product over time. We also track campaign and referral data on our mobile app to help { -brand-name-mozilla } understand the effectiveness of our marketing campaigns. Read more in our <a { $notice }>Privacy Notice</a>.
+
+vpn-pricing-how-do-i-manage = How do I manage my subscription and change my plan?
+
+# Variables
+# $manage (url) - link to subscription management page
+vpn-pricing-if-already-subscribed = If you’re already subscribed to { -brand-name-mozilla-vpn }, you can change your plan or <a { $manage }">manage your subscription</a> anytime.
```
Comment (alexgibson) [inline l10n/en/products/vpn/pricing-2023.ftl:48]: Nit: remove extra `"` from the link here?
```suggestion
vpn-pricing-if-already-subscribed = If you’re already subscribed to { -brand-name-mozilla-vpn }, you can change your plan or <a { $manage }>manage your subscription</a> anytime.
```

**Review by alexgibson — CHANGES_REQUESTED** [review]
A few nits and things to fix up, but otherwise this all looks good to me.

One other thing that we should address in this PR is update the functional tests. Now that the pages are being localized, we can't be sure which locales will see the old page, so the existing tests for `/de/` will start to fail:

- https://github.com/mozilla/bedrock/blob/main/tests/functional/products/vpn/test_landing.py
- https://github.com/mozilla/bedrock/blob/main/tests/functional/products/vpn/test_pricing.py

Rather than delete them now (which is maybe a bit to early?), I wonder if we could add some logic in the view so that the old template can be viewed via a special param, such as `xv=legacy`?  Then the tests below could include that param when they run, so we still get coverage for the old page. 

Thoughts? Does that make sense?

File: bedrock/products/views.py (hunk 2 above)
Comment (reemhamz) [reply bedrock/products/views.py:106]: Seems like we don't -- removing. Thanks for the eagle eyes!

**Review by alexgibson — CHANGES_REQUESTED** [review]
Tests for the old templates still need updating to use the new `xv=legacy` param:

- https://github.com/mozilla/bedrock/blob/main/tests/functional/products/vpn/test_landing.py
- https://github.com/mozilla/bedrock/blob/main/tests/functional/products/vpn/test_pricing.py

**Comment by craigcook** [comment]
Updated tests, and also squashed and rebased.

**Review by alexgibson — APPROVED** [review]
r+

**Pull request merged**